argostime/static/table-sort.js

```diff
@@ -0,0 +1,371 @@
+/* 
+table-sort-js
```

Comment:
```suggestion
table-sort-js 1.6.8
```
To make clear which version we're using so that we can easily see if we need to update it